src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameControl.xaml

```diff
@@ -0,0 +1,91 @@
+﻿<UserControl x:Class="Microsoft.CodeAnalysis.InlineRename.UI.SmartRename.SmartRenameControl"
+             x:ClassModifier="internal"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:vsfx="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+             xmlns:platformUI="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+             mc:Ignorable="d"
+             Margin="0 0 0 5">
+    <UserControl.Resources>
+        <BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
+        <SolidColorBrush x:Key="copilotBranding" Color="#6F66E3"/>
+
+        <Style x:Key="CopilotListViewStyle" TargetType="ListView">
+            <Setter Property="ItemsSource" Value="{Binding SuggestedNames}"/>
+            <Setter Property="Visibility" Value="{Binding HasSuggestions, Converter={StaticResource BooleanToVisibilityConverter}, Mode=OneWay}"/>
+            <Setter Property="Background" Value="Transparent"/>
+            <Setter Property="BorderThickness" Value="0"/>
+            <Setter Property="BorderBrush" Value="{StaticResource copilotBranding}"/>
+
+            <Setter Property="ItemsPanel">
+                <Setter.Value>
+                    <ItemsPanelTemplate>
+                        <WrapPanel Orientation="Horizontal" Width="238" HorizontalAlignment="Left"/>
```

Comment:
slightly concerned about hardcoded values like 238.  where did that number come from?

Reply:
I got this from the editor side impl, tag @AmadeusW here

Reply:
I'd try it without tbh. It should fill the space of the panel

Reply:
Without the width limitation, it will try to put everything in the same line
![image](https://github.com/dotnet/roslyn/assets/24360909/039dcb68-fd0f-41fd-815a-f4a23099be88)